### PR TITLE
en: add a doc and notes about deploying a cluster on ARM64

### PR DESCRIPTION
**Pull request by en-jin19**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
Added a new document and notes about deploying a cluster on ARM64.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1301
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by KanShiori — APPROVED** [review]

File: en/deploy-cluster-on-arm64.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+title: Deploy a TiDB Cluster on ARM64 Machine
+summary: Learn how to deploy a TiDB cluster on ARM64 Machine.
+---
+
+# Deploy a TiDB Cluster on ARM64 Machine
+
+This document describes how to deploy a TiDB cluster on ARM64 machine.
```
Comment (qiancai) [inline en/deploy-cluster-on-arm64.md:8]:
```suggestion
This document describes how to deploy a TiDB cluster on ARM64 machines.
```

File: en/deploy-cluster-on-arm64.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+## Prerequisites
+
+Before starting the process, make sure Kubernetes is deployed on your ARM64 machine. If Kubernetes is not deployed, refer to [Deploy the Kubernetes cluster](deploy-tidb-operator.md#deploy-the-kubernetes-cluster).
```
Comment (qiancai) [inline en/deploy-cluster-on-arm64.md:12]:
```suggestion
Before starting the process, make sure that Kubernetes clusters are deployed on your ARM64 machines. If Kubernetes clusters are not deployed, refer to [Deploy the Kubernetes cluster](deploy-tidb-operator.md#deploy-the-kubernetes-cluster).
```

File: en/deploy-cluster-on-arm64.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+## Deploy TiDB operator
+
+The process of deploying TiDB operator on ARM64 machine is the same as the process of [Deploy TiDB Operator in Kubernetes](deploy-tidb-operator.md). The only difference is the following part in the step [Customize TiDB operator deployment](deploy-tidb-operator.md#customize-tidb-operator-deployment): after getting the `values.yaml` file of the `tidb-operator` chart, you need to modify the fields of `operatorImage` and `tidbBackupManagerImage` in that file to the image of ARM64 version. For example:
```
Comment (qiancai) [inline en/deploy-cluster-on-arm64.md:16]:
```suggestion
The process of deploying TiDB operator on ARM64 machines is the same as the process of [Deploy TiDB Operator in Kubernetes](deploy-tidb-operator.md). The only difference is the following configuration in the step [Customize TiDB operator deployment](deploy-tidb-operator.md#customize-tidb-operator-deployment): after getting the `values.yaml` file of the `tidb-operator` chart, you need to modify the `operatorImage` and `tidbBackupManagerImage` fields in that file to the ARM64 image versions. For example:
```

File: en/deploy-cluster-on-arm64.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+# ...
+operatorImage: pingcap/tidb-operator-arm64:v1.2.4
+# ...
+tidbBackupManagerImage: pingcap/tidb-backup-manager-arm64:v1.2.4
+# ...
+```
+
+## Deploy a TiDB cluster
+
+The process of deploying a TiDB cluster on ARM64 machine is the same as the process of [Deploy TiDB in General Kubernetes](deploy-on-general-kubernetes.md). The only difference is that you need to set the images of the related components in the TidbCluster definition file to ARM64 version. For example:
```
Comment (qiancai) [inline en/deploy-cluster-on-arm64.md:28]:
```suggestion
The process of deploying a TiDB cluster on ARM64 machines is the same as the process of [Deploy TiDB in General Kubernetes](deploy-on-general-kubernetes.md). The only difference is that, in the TidbCluster definition file, you need to set the images of the related components to the ARM64 versions. For example:
```

File: en/deploy-cluster-on-arm64.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    baseImage: pingcap/ticdc-arm64
+    # ...
+  tiflash:
+    baseImage: pingcap/tiflash-arm64
+    # ...
+```
+
+## Initialize a TiDB Cluster
+
+The process of initializing a TiDB cluster on ARM64 machine is the same as the process of [Initialize a TiDB Cluster in Kubernetes](initialize-a-cluster.md). The only difference is that you need to modify the field `spec.image` in the TidbInitializer definition file to the image of ARM64 version. For example:
```
Comment (qiancai) [inline en/deploy-cluster-on-arm64.md:64]:
```suggestion
The process of initializing a TiDB cluster on ARM64 machines is the same as the process of [Initialize a TiDB Cluster in Kubernetes](initialize-a-cluster.md). The only difference is that you need to modify the `spec.image` field in the TidbInitializer definition file to the ARM64 image version. For example:
```

Comment (qiancai) [reply en/deploy-cluster-on-arm64.md:64]: the field xxx > the xxx field

File: en/deploy-cluster-on-arm64.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+    # ...
+  ticdc:
+    baseImage: pingcap/ticdc-arm64
+    # ...
+  tiflash:
+    baseImage: pingcap/tiflash-arm64
+    # ...
+```
+
+## Initialize a TiDB Cluster
```
Comment (qiancai) [inline en/deploy-cluster-on-arm64.md:62]:
```suggestion
## Initialize a TiDB cluster
```

File: en/deploy-cluster-on-arm64.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+  name: ${initializer_name}
+  namespace: ${cluster_namespace}
+spec:
+  image: kanshiori/mysqlclient-arm64
+  # ...
+```
+
+## Deploy monitoring for a TiDB cluster
+
+The process of deploying monitoring for a TiDB cluster on ARM64 machine is the same as the process of [Deploy Monitoring and Alerts for a TiDB Cluster](monitor-a-tidb-cluster.md). The only difference is that you need to modify the fields of `spec.initializer.baseImage` and `spec.reloader.baseImage` in the TidbMonitor definition file to the image of ARM64 version.
```
Comment (qiancai) [inline en/deploy-cluster-on-arm64.md:79]:
```suggestion
The process of deploying monitoring for a TiDB cluster on ARM64 machines is the same as the process of [Deploy Monitoring and Alerts for a TiDB Cluster](monitor-a-tidb-cluster.md). The only difference is that you need to modify the `spec.initializer.baseImage` and `spec.reloader.baseImage` fields in the TidbMonitor definition file to the ARM64 image versions.
```

File: en/deploy-on-alibaba-cloud.md [hunk 8]
```diff
@@ -264,6 +264,10 @@ All the instances except ACK mandatory workers are deployed across availability
     kubectl --kubeconfig credentials/kubeconfig create -f db-monitor.yaml -n ${namespace}
     ```
 
+> **Note:**
+>
+> If you need to deploy a TiDB cluster on ARM64 machine, refer to [Deploy a TiDB Cluster on ARM64 Machine](deploy-cluster-on-arm64.md).
```
Comment (qiancai) [inline en/deploy-on-alibaba-cloud.md:269]:
```suggestion
> If you need to deploy a TiDB cluster on ARM64 machines, refer to [Deploy a TiDB Cluster on ARM64 Machines](deploy-cluster-on-arm64.md).
```

File: en/deploy-on-aws-eks.md [hunk 9]
```diff
@@ -196,6 +196,10 @@ kubectl apply -f tidb-monitor.yaml -n tidb-cluster
 
 After the yaml file above is applied to the Kubernetes cluster, TiDB Operator creates the desired TiDB cluster and its monitoring component according to the yaml file.
 
+> **Note:**
+>
+> If you need to deploy a TiDB cluster on ARM64 machine, refer to [Deploy a TiDB Cluster on ARM64 Machine](deploy-cluster-on-arm64.md).
```
Comment (qiancai) [inline en/deploy-on-aws-eks.md:201]:
```suggestion
> If you need to deploy a TiDB cluster on ARM64 machines, refer to [Deploy a TiDB Cluster on ARM64 Machines](deploy-cluster-on-arm64.md).
```

File: en/deploy-on-gcp-gke.md [hunk 10]
```diff
@@ -113,6 +113,10 @@ kubectl create -f tidb-monitor.yaml -n tidb-cluster
 
 After the yaml file above is applied to the Kubernetes cluster, TiDB Operator creates the desired TiDB cluster and its monitoring component according to the yaml file.
 
+> **Note:**
+>
+> If you need to deploy a TiDB cluster on ARM64 machine, refer to [Deploy a TiDB Cluster on ARM64 Machine](deploy-cluster-on-arm64.md).
```
Comment (qiancai) [inline en/deploy-on-gcp-gke.md:118]:
```suggestion
> If you need to deploy a TiDB cluster on ARM64 machines, refer to [Deploy a TiDB Cluster on ARM64 Machines](deploy-cluster-on-arm64.md).
```

File: en/deploy-on-general-kubernetes.md [hunk 11]
```diff
@@ -118,6 +118,10 @@ You can use TiDB Operator to deploy and manage multiple TiDB clusters in a singl
 
 Different clusters can be in the same or different `namespace`, which is based on your actual needs.
 
+> **Note:**
+>
+> If you need to deploy a TiDB cluster on ARM64 machine, refer to [Deploy a TiDB Cluster on ARM64 Machine](deploy-cluster-on-arm64.md).
```
Comment (qiancai) [inline en/deploy-on-general-kubernetes.md:123]:
```suggestion
> If you need to deploy a TiDB cluster on ARM64 machines, refer to [Deploy a TiDB Cluster on ARM64 Machines](deploy-cluster-on-arm64.md).
```

File: en/get-started.md [hunk 12]
```diff
@@ -380,6 +380,10 @@ namespace/tidb-cluster created
 tidbcluster.pingcap.com/basic created
 ```
 
+> **Note:**
+>
+> If you need to deploy a TiDB cluster on ARM64 machine, refer to [Deploy a TiDB Cluster on ARM64 Machine](deploy-cluster-on-arm64.md).
```
Comment (qiancai) [inline en/get-started.md:385]:
```suggestion
> If you need to deploy a TiDB cluster on ARM64 machines, refer to [Deploy a TiDB Cluster on ARM64 Machines](deploy-cluster-on-arm64.md).
```

**Review by qiancai — COMMENTED** [review]
@en-jin19 I've completed the review and left some comments.  PTAL. Thanks.

**Review by qiancai — APPROVED** [review]

**Pull request merged**